python/lsst/ts/Dome/mock_llc_statuses/amcs_status.py

```diff
@@ -55,23 +54,34 @@ async def determine_status(self, current_tai):
         )
         if self.status != LlcStatus.STOPPED.value:
             azimuth_step = self.motion_velocity * time_diff
-            if self.motion_direction == motion_dir.CW.value:
-                self.position_actual = self.position_orig + azimuth_step
+            self.position_actual = self.position_orig + azimuth_step
+            if self.motion_velocity >= 0:
                 if self.position_actual >= self.position_cmd:
                     self.position_actual = self.position_cmd
-                    self.position_orig = self.position_actual
                     if self.status == LlcStatus.MOVING.value:
-                        self.status = LlcStatus.STOPPED.value
+                        if self.crawl_velocity >= 0:
+                            self.position_cmd = math.radians(720)
+                        else:
+                            self.position_cmd = math.radians(-360)
```

Comment:
Some explanation of these values would be appreciated. I think you mean these to be "always larger or smaller than the current position". If math.inf would work it might add clarity and eliminate a possible error if you leave the dome crawling for a long time (though you'd eventually run into inaccuracy due to rounding large angles).

Reply:
I had math.inf (and -math.inf) before and decided to replace it. I don't remember why. I have put math.inf back with an inline comment.